src/components/Link/tests/Link.test.tsx

```diff
@@ -123,4 +123,32 @@ describe('<Link />', () => {
       });
     });
   });
+
+  describe('accessibilityLabel', () => {
+    it('passes prop to the button url is not provided', () => {
+      const mockAccessibilityLabel = 'mock accessibility label';
+      const link = mountWithAppProvider(
+        <Link accessibilityLabel={mockAccessibilityLabel} />,
+      );
+
+      expect(link.find('button').prop('aria-label')).toBe(
```

Comment:
We'll have to fix our expect line once we've made the change with mountWithApp. We may want to say something like: expect(link).toContainReactComponent('button', {'aria-label': mockAccessibilityLabel, });